RecoBTag/Configuration/python/RecoBTag_cff.py

```diff
@@ -65,8 +66,11 @@
     # overall combined taggers
     #CSV + soft-lepton + jet probability discriminators combined
     pfCombinedMVAV2BJetTags,
-    pfChargeBJetTags
+    pfChargeBJetTags,
+    # new DeepFlavour tagger
```

Comment:
"new" could probably be dropped

Reply:
the reason for the new is that the DeepCSV+DeepCMVA task by @mverzett is also called DeepFlavourTask (check https://github.com/pablodecm/cmssw/blob/ffecd656824a92a6b2cb5cd0c81dfb707aee0ae2/RecoBTag/Combined/python/deepFlavour_cff.py#L53 and https://github.com/pablodecm/cmssw/blob/ffecd656824a92a6b2cb5cd0c81dfb707aee0ae2/RecoBTag/Configuration/python/RecoBTag_cff.py#L58).
@slava77 shoud we rename the old DeepFlavourTask or something else?

Reply:
@pablodecm please do, call the old one DeepCSVTask, thanks

Reply:
@mverzett  done in https://github.com/cms-sw/cmssw/pull/19893/commits/9ec8d77fee43c3f2a8064a224eed83aa52a11fb2 and removed New postfix